app/src/main/java/com/droidhats/campuscompass/views/ExploreFragment.kt

```diff
@@ -0,0 +1,73 @@
+package com.droidhats.campuscompass.views
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.Button
+import android.widget.ImageButton
+import android.widget.ToggleButton
+import androidx.core.view.GravityCompat
+import androidx.drawerlayout.widget.DrawerLayout
+import androidx.fragment.app.Fragment
+import com.droidhats.campuscompass.R
+import androidx.navigation.fragment.findNavController
+import com.google.android.gms.maps.model.LatLng
+import kotlinx.android.synthetic.main.search_bar_layout.*
+
+class ExploreFragment: Fragment() {
+
+    private lateinit var root : View
```

Comment:
I would suggest that we use remove the extra space after `root` so that we follow Kotlin's coding convention when declaring variable types. It can be like: `private lateinit var root: View`

Reply:
Fixed